### ISSUE TITLE
WQ_POWER_EFFICIENT_DEFAULT

### ISSUE THREAD

**Issue by wojtoo** [blankaf/kernel_xiaomi_msm8998#2]
blankaf
if you added power_efficient_wq to kernel please consider CONFIG_WQ_POWER_EFFICIENT_DEFAULT=y in sagit_config
https://patchwork.kernel.org/patch/9878679/
thanks :)

**Comment by blankaf** [comment]
Yeah, I am gonna do that. I just haven't gotten there yet. Trying to fix charging.
